### PR TITLE
UHM-4791 Migrate "Migrate Infants" to SQL Migrator

### PR DESCRIPTION
**Pull request by brandones**
https://pihemr.atlassian.net/browse/UHM-4791

Will change PR to be against `master` once https://github.com/PIH/hivmigration/pull/10 is merged.

### CODE REVIEW

**Comment by mogoodrich** [comment]
Thanks @brandones , I'll take a look tomorrow...

File: etl/src/main/java/org/pih/hivmigration/etl/sql/InfantMigrator.groovy [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+        executeMysql("Add UUIDs", "update hivmigration_infants set person_uuid = uuid();")
+
+        executeMysql("Load to person table", '''
+            insert into person
+              (person_id, uuid, gender, birthdate, creator, date_created)
+            select
+              i.person_id,
+              i.person_uuid,
+              COALESCE(i.gender, 'U'),  # most of the data is missing gender -- default to U
```
Comment (mogoodrich) [inline etl/src/main/java/org/pih/hivmigration/etl/sql/InfantMigrator.groovy:56]: I forget, does OpenMRS now support this? (U for gender?)

File: etl/src/main/java/org/pih/hivmigration/etl/sql/InfantMigrator.groovy [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+              i.person_id,
+              1,
+              date_format(curdate(), '%Y-%m-%d %T')
+            from
+              hivmigration_infants i
+            order by i.source_infant_id;
+        ''')
+
+        // TODO: depends on zlemrid from Patient migration (https://pihemr.atlassian.net/browse/UHM-3145)
+        /*
```
Comment (mogoodrich) [inline etl/src/main/java/org/pih/hivmigration/etl/sql/InfantMigrator.groovy:93]: Cool, I can take care of this when I finish Patients (hopefully today)... can you assign this ticket to me once you merge to remind me to take care of this?

File: etl/src/main/java/org/pih/hivmigration/etl/sql/InfantMigrator.groovy (hunk 1 above)
Comment (brandones) [reply etl/src/main/java/org/pih/hivmigration/etl/sql/InfantMigrator.groovy:56]: No, but it doesn't break anything, while NULL does. It just shows "U" in the places you would expect a gender. There are even some message codes that you can define with it.

This is how I migrated missing genders for Mexico; no regrets yet!

File: etl/src/main/java/org/pih/hivmigration/etl/sql/InfantMigrator.groovy (hunk 2 above)
Comment (brandones) [reply etl/src/main/java/org/pih/hivmigration/etl/sql/InfantMigrator.groovy:93]: Sure thing, thanks!

**Pull request merged**